### PR TITLE
Mosh all the open NAS PRs together

### PR DESCRIPTION
**Pull request by Shredder121**
https://github.com/Shredder121-me/jda-nas/commit/50ea2637cdc8fc63970c3cb51ae3e6e851ec06ec

https://github.com/Shredder121-me/jda-nas/compare/master...Shredder121-me:mosh

I think this is reasonably done, although we could depend on  `jda-nas` anyway, in the off chance we're missing some dependencies.

### CODE REVIEW

File: FredBoat/pom.xml [hunk 1]
```diff
@@ -75,9 +75,16 @@
...
+            <groupId>com.sedmelluq</groupId>
+            <artifactId>lavaplayer-common</artifactId>
+            <version>1.0.5</version>
+        </dependency>
+        <dependency>
             <!-- native audio system to avoid gc stutter in the audio -->
+            <!-- we replaced the classes temporarily, but need the native parts -->
             <groupId>com.sedmelluq</groupId>
-            <artifactId>jda-nas</artifactId>
+            <artifactId>udp-queue</artifactId>
```
Comment (Shredder121) [inline FredBoat/pom.xml]: yeah this is probably not enough.

**Comment by napstr** [comment]
Is this intended to go into FredBoat codebase long term, or is it just a temporary hack?

**Comment by Shredder121** [comment]
https://github.com/Repulser/jda-nas/commit/ec24b5cd7229abf9e39e4460b2fa73daf31f69f5
Repulser built a version of jda-nas with the outstanding PRs.
So we could depend on
```xml
<dependency>
  <groupId>com.github.shredder121.fork</groupId>
  <artifactId>com.sedmelluq.jda-nas</artifactId>
  <version>1.0.8-rythm</version>
</dependency>
```
And add the repository `https://jcenter.bintray.com/com/github/shredder121/`

Usually I like to have it in Central, but jda-nas doesn't produce javadoc.

**Comment by Frederikam** [comment]
Merged for now. We can always depend on the new artefact.

**Comment by Shredder121** [comment]
Ah, ok fair enough

**Pull request merged**